docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+.. currentmodule:: discord
+
+
+==========================
+Errors and Error Handling
+==========================
+
+Handling Errors
+===============
+
+When an error ocurrs in a command or an event, it will, by default, show an error in the console to denote that the error 
+ocurred and has been silently ignored. In order to avoid this, an error handler is needed.
+
+In :ref:`Commands <discord_ext_commands>`
+-----------------------------------------
+
+All errors that are raised by a command derive from :exc:`~.ext.commands.CommandError`. If an error that occurs during a command does
```

Comment:
wouldn't this be better after after showing global error handlers as stuff?, as in this case i might have no clue how i got the "error" here or where it's showing up, which might confuse people